webapp/portlet/src/main/webapp/vue-apps/common/components/ExoSpaceAvatar.vue

```diff
@@ -9,16 +9,17 @@
       class="flex-nowrap flex-shrink-0 d-flex spaceAvatar not-clickable-link">
       <v-avatar
         :size="size"
+        :class="pullLeft"
```

Comment:
Does this will change behaviour from ExoSpaceAvatar when used with mobile in another component than ActivityHead ?